.mergify.yml

```diff
@@ -15,86 +15,6 @@ pull_request_rules:
-    conditions:
-      - author=mergify[bot]
-      - base=release-1.8
-      - label!=do-not-merge
-      - "status-success=DCO"
-      - "check-success=canary"
-      - "check-success=unittests"
-      - "check-success=golangci-lint"
-      - "check-success=codegen"
-      - "check-success=lint"
-      - "check-success=modcheck"
-      - "check-success=pvc"
-      - "check-success=pvc-db"
-      - "check-success=pvc-db-wal"
-      - "check-success=encryption-pvc"
-      - "check-success=encryption-pvc-db"
-      - "check-success=encryption-pvc-db-wal"
-      - "check-success=encryption-pvc-kms-vault-token-auth"
-      - "check-success=encryption-pvc-kms-vault-k8s-auth"
-      - "check-success=lvm-pvc"
-      - "check-success=multi-cluster-mirroring"
-      - "check-success=rgw-multisite-testing"
-      - "check-success=TestCephSmokeSuite (v1.16.15)"
-      - "check-success=TestCephSmokeSuite (v1.22.2)"
-      - "check-success=TestCephHelmSuite (v1.16.15)"
-      - "check-success=TestCephHelmSuite (v1.22.2)"
-      - "check-success=TestCephMultiClusterDeploySuite (v1.22.2)"
-      - "check-success=TestCephUpgradeSuite (v1.16.15)"
-      - "check-success=TestCephUpgradeSuite (v1.22.2)"
-    actions:
-      merge:
-        method: merge
-      dismiss_reviews: {}
-      delete_head_branch: {}
-
-  # release-1.9 branch
-  - name: automerge backport release-1.9
-    conditions:
-      - author=mergify[bot]
-      - base=release-1.9
-      - label!=do-not-merge
-      - "status-success=DCO"
-      - "check-success=canary"
-      - "check-success=unittests"
-      - "check-success=golangci-lint"
-      - "check-success=codegen"
-      - "check-success=codespell"
-      - "check-success=lint"
-      - "check-success=modcheck"
-      - "check-success=Shellcheck"
-      - "check-success=yaml-linter"
-      - "check-success=lint-test"
-      - "check-success=gen-rbac"
-      - "check-success=crds-gen"
-      - "check-success=pvc"
-      - "check-success=pvc-db"
-      - "check-success=pvc-db-wal"
-      - "check-success=encryption-pvc"
-      - "check-success=encryption-pvc-db"
-      - "check-success=encryption-pvc-db-wal"
-      - "check-success=encryption-pvc-kms-vault-token-auth"
-      - "check-success=encryption-pvc-kms-vault-k8s-auth"
-      - "check-success=lvm-pvc"
-      - "check-success=multi-cluster-mirroring"
-      - "check-success=rgw-multisite-testing"
-      - "check-success=TestCephSmokeSuite (v1.17.17)"
-      - "check-success=TestCephSmokeSuite (v1.23.0)"
-      - "check-success=TestCephHelmSuite (v1.17.17)"
-      - "check-success=TestCephHelmSuite (v1.23.0)"
-      - "check-success=TestCephMultiClusterDeploySuite (v1.23.0)"
-      - "check-success=TestCephUpgradeSuite (v1.17.17)"
-      - "check-success=TestCephUpgradeSuite (v1.23.0)"
-    actions:
-      merge:
-        method: merge
-      dismiss_reviews: {}
-      delete_head_branch: {}
-
   # release-1.10 branch
   - name: automerge backport release-1.10
```

Comment:
not for 1.14?

Reply:
The section for 1.14 is at the end of the file